README.md

```diff
@@ -270,11 +271,18 @@ _Mass mailing software - one message to many recipients_
 
 *[SIP](https://en.wikipedia.org/wiki/Session_Initiation_Protocol) telephony software*
 
-  * [Asterisk](http://www.asterisk.org/) - An easy to use but advanced IP PXB system, VoIP gateway and conference server. `GPLv2` `C`
+  * [Asterisk](http://www.asterisk.org/) - An easy to use but advanced IP PBX system, VoIP gateway and conference server. `GPLv2` `C`
   * [FreeSWITCH](https://freeswitch.org/) - Scalable open source cross-platform telephony platform. ([Source Code](https://freeswitch.org/stash/projects/FS/repos/freeswitch/browse)) `MPLv2` `C`
   * [Kamailio](http://www.kamailio.org/w/) - Modular SIP server (registrar/proxy/router/etc) ([Source Code](https://github.com/kamailio/kamailio)) `GPLv2` `C`
   * [Ostel](https://dev.guardianproject.info/projects/ostel/wiki/Server_Documentation) - Secure SIP telephony setup with ZRTP encryption. `GPLv3` `Ruby`
 
+### IPBX
+
+ *[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software*
+ 
+  * [Elastix](http://www.elastix.org) - Unified communications server software based on Asterisk ([Source Code](https://svn.code.sf.net/p/elastix/code/trunk/)) `GPLv2` `PHP`
+  * [Freepbx](http://www.freepbx.org) - Web-based open source GUI (graphical user interface) that controls and manages Asterisk ([Source Code](http://git.freepbx.org/projects/FREEPBX)) `GPLv2` `PHP`
```

Comment:
Should be

    [Freepbx](http://www.freepbx.org) - Web-based open source GUI (graphical user interface) that controls and manages Asterisk. ([Source Code](http://git.freepbx.org/projects/FREEPBX)) `GPLv2` `PHP`